db/migrate/20150418001125_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :password
+      t.string :email
+      t.string :first_name
+      t.string :phone_number
+      t.string :role
```

Comment:
Minnie and I will be implementing that this week and because this will really be Minnie's first time I will let decide which way to do it. But the reason I added them was because at the first meeting we decide on these columns. But they are super easy to change if at some point we need to.